CHANGELOG.md

```diff
@@ -7,9 +7,20 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+**CriticalUp is now travel and spotty internet friendly!** New retry and caching behavior, as well as
+an `--offline` mode for `criticalup install` means downloading new packages should be more reliable,
+and packages should not need to be redownloaded when used between multiple projects or toolchain
+reinstallation. *Now you can finally write automotive software while on a road trip into a remote part
+of the Pacific coast of Canada!*
```

Comment:
Writing to appreciate the level of detail in bringing in the use cases. :heart:

Reply:
Unsure if release notes of this candor are appropriate here, to be honest. 😅